x-pack/plugins/ml/public/application/settings/filter_lists/edit/edit_filter_list.js

```diff
@@ -320,6 +322,8 @@ export class EditFilterListUI extends Component {
 
     const totalItemCount = items !== undefined ? items.length : 0;
 
+    const helpLink = getDocLinks().links.ml.guide;
```

Comment:
Is it worth linking to the Custom rules [page](https://www.elastic.co/guide/en/machine-learning/master/ml-rules.html) here rather than the top level of the ML guide?  If so, looks like you'd need to add that custom rules URL to the doc links service.